app/assets/javascripts/discourse/admin/components/admin-config-areas/upcoming-changes.gjs

```diff
@@ -0,0 +1,134 @@
+import Component from "@glimmer/component";
```

Comment:
Note I haven't done a system spec or component spec on this and upcoming-change-item for now because this UI still may be in flux, and it's behind a flag. I will do that in a followup PR